<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>401</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>SEcube Login</string>
    </property>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>show password</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>381</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>Access mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_username">
        <item>
         <property name="text">
          <string>Admin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>User</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>381</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>SEcube PIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced options</string>
    </property>
    <addaction name="actionInitialize_admin_SEcube"/>
    <addaction name="separator"/>
    <addaction name="actionSet_SEkey_update_folder"/>
   </widget>
   <addaction name="menuAdvanced"/>
  </widget>
  <action name="actionInitialize_admin_SEcube">
   <property name="text">
    <string>Initialize admin SEcube</string>
   </property>
  </action>
  <action name="actionSet_SEkey_update_folder">
   <property name="text">
    <string>Set SEkey update folder</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_username</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>lineEdit_password</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
